edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss

```diff
@@ -1,7 +1,358 @@
-// Extra rules needed for an LMS theme
+// Extra rules needed for the LMS dark theme
 //
 // The _extras.scss file is imported after all other rules by
 // the LMS Sass. For your own theme, you can override this file
 // to add any custom rules that you need. You can also import
 // partials directly into this file so that you can break your
 // rules into modular pieces.
+
+@import 'certificates';
+@import 'courses';
+@import 'header';
+@import 'navigation';
+@import 'footer';
+@import 'profile';
+@import 'index';
+@import 'dashboard';
+@import 'support';
+@import 'course-experience';
+
+
+//registration
+.view-register .form-actions button[type="submit"]:hover {
+  background-color:#fff;
+  border: 1px solid #007FA3;
+}
+
+
```

Comment:
There are two empty lines between the class definitions in the whole file, can we leave a single line? Instead of two ... @luismorenolopera